backend/src/database/daos/posts/posts.dao.service.ts

```diff
@@ -86,7 +86,8 @@ export class PostsDaoService {
         } else if (value === PostType.SELLER) {
           query.where('type', PostType.BUYER);
         } else {
-          query.where('type', PostType.OTHER);
+          // PostType.OTHERS: return posts that have 'open to climb with others'
+          query.where('openToClimbTogether', true);
```

Comment:
I was thinking we don't need this condition.

Reply:
update: agreed to leave this in till we improve on the searching parameters next sprint